src/wrapping.rs

```diff
@@ -63,6 +70,68 @@ impl<T: ConstantTimeEq> ConstantTimeEq for Wrapping<T> {
     }
 }
 
+#[cfg(feature = "num-traits")]
+impl<T: Zero> num_traits::Zero for Wrapping<T>
+    where Wrapping<T>:
+    Add<Wrapping<T>, Output = Wrapping<T>>
+    + PartialEq<Wrapping<T>>
+{
+    fn zero() -> Self {
+        Self::ZERO
+    }
+
+    fn is_zero(&self) -> bool { self == &Self::ZERO }
+}
+
+#[cfg(feature = "num-traits")]
+impl<T: Integer> num_traits::One for Wrapping<T>
+    where Wrapping<T>:
+    Add<Wrapping<T>, Output = Wrapping<T>>
+    + Mul<Wrapping<T>, Output = Wrapping<T>>
+{
+    fn one() -> Self {
+        Wrapping(T::ONE)
+    }
+}
+
+#[cfg(feature = "num-traits")]
+impl<T: Zero> Div<Wrapping<T>> for Wrapping<T>
+    where Wrapping<T>: Div<NonZero<T>, Output = Wrapping<T>>
+{
+    type Output = Wrapping<T>;
+
+    fn div(self, rhs: Wrapping<T>) -> Self::Output {
+        self / NonZero::new(rhs.0).unwrap()
+    }
+}
+
+#[cfg(feature = "num-traits")]
+impl<T: Zero> Rem<Wrapping<T>> for Wrapping<T>
+    where Wrapping<T>: Rem<NonZero<T>, Output = Wrapping<T>>
+{
+    type Output = Wrapping<T>;
+
+    fn rem(self, rhs: Wrapping<T>) -> Self::Output {
+        self % NonZero::new(rhs.0).unwrap()
+    }
+}
+
+#[cfg(feature = "num-traits")]
+impl<T: Integer + Zero> Num for Wrapping<T>
+    where Wrapping<T>:
+    Add<Wrapping<T>, Output = Wrapping<T>>
+    + Sub<Wrapping<T>, Output = Wrapping<T>>
+    + Mul<Wrapping<T>, Output = Wrapping<T>>
+    + Div<NonZero<T>, Output = Wrapping<T>>
+    + Rem<NonZero<T>, Output = Wrapping<T>>
+{
+    type FromStrRadixErr = ();
+
+    fn from_str_radix(str: &str, radix: u32) -> Result<Self, Self::FromStrRadixErr> {
```

Comment:
How do we want to handle this?

Obviously this is unimplemented logic for this crate, and I'm not sure whether we want to support this (or, for example, return an error for any radix other than 2, 10 or 16.)

`num-bigint`'s [implementation](https://github.com/rust-num/num-bigint/blob/6f2b8e0fc218dbd0f49bebb8db2d1a771fe6bafa/src/biguint/convert.rs#LL220C75-L220C75) is complex, but I could copy it if that's the way to go.

My thoughts are to actually to use `from_str_radix()` implementation of [`u32`](https://doc.rust-lang.org/std/primitive.u32.html) and [`u64`](https://doc.rust-lang.org/std/primitive.u64.html), that is, to parse limb-by-limb and sum them up.

Reply:
#246 and/or #247 are some options, although I took issue with them not being constant-time